ps_imageslider.php

```diff
@@ -432,6 +443,18 @@ protected function _postProcess()
             $res = $slide->update();
             $this->clearCache();
             $this->_html .= ($res ? $this->displayConfirmation($this->getTranslator()->trans('Configuration updated', array(), 'Admin.Notifications.Success')) : $this->displayError($this->getTranslator()->trans('The configuration could not be updated.', array(), 'Modules.Imageslider.Admin')));
+        
+        } elseif (Tools::isSubmit('changeMobile') && Tools::isSubmit('id_slide')) {
+            $slide = new Ps_HomeSlide((int)Tools::getValue('id_slide'));
+            if ($slide->mobile == 0) {
+                $slide->mobile = 1;
+            } else {
+                $slide->mobile = 0;
+            }
```

Comment:
```suggestion
            $slide->mobile = $slide->mobile == 0 ? 1 : 0;
```

Reply:
I don't get it, so whatever you set, it's always 1/true/value :thinking: